src/msw/window.cpp

```diff
@@ -6276,6 +6331,40 @@ bool wxWindowMSW::HandlePressAndTap(const wxPoint& pt, WXDWORD flags)
 
     return HandleWindowEvent(event);
 }
+
+bool wxWindowMSW::HandleTouch(WXWPARAM wParam, WXLPARAM lParam)
+{
+    const unsigned count = LOWORD(wParam);
+    wxVector<TOUCHINPUT> info(count);
+    if (GestureFuncs::GetTouchInputInfo() ((HTOUCHINPUT)lParam, count, &info[0], sizeof(TOUCHINPUT)))
+    {
+        for(unsigned i = 0; i < count; i++)
```

Comment:
```suggestion
        for ( const auto& input : info )
```
(and then use `input` instead of `info[i]` in the loop body, of course)